core/test/results/sample-config.js

```diff
@@ -13,37 +13,6 @@ const budgetedConfig = {
   extends: 'lighthouse:default',
   settings: {
     throttlingMethod: 'devtools',
-    budgets: [{
```

Comment:
nit: this is called `budgetedConfig` here and in `update-report-fixtures.js`. Could we just call it `sampleConfig`?